tests/trainer/test_checkpoint.py

```diff
@@ -118,6 +118,14 @@ def _assert_checkpoints_equivalent(file1, file2, atol=0.0, rtol=0.0):
     del checkpoint_1['state']['callbacks']['CheckpointSaver']['all_saved_checkpoints_to_timestamp']
     del checkpoint_2['state']['callbacks']['CheckpointSaver']['all_saved_checkpoints_to_timestamp']
 
+    # Remove verbose label in state/schedulers if exists
```

Comment:
what is this?

Reply:
Was trying to get rid of this error: https://github.com/mosaicml/composer/actions/runs/15181682096/job/42692335913 but this doesn't seem to resolve it; will figure out a better solution today

Reply:
Moved it to a better location